build.sbt

```diff
@@ -36,3 +36,5 @@ val partestSuite = Build.partestSuite
 val scalaTestSuite = Build.scalaTestSuite
 
 inThisBuild(Build.thisBuildSettings)
+
+addCommandAlias("scalastyle", "; scalastyleCheck")
```

Comment:
I'm not sure this indirection is necessary. Invoking `scalastyleCheck` is just as fine, no?

Reply:
`scalastyle` is the usual command people run, so it's a bit more "friendly", but no, it is not necessary, thoughts?  I originally had just `scalastyleCheck` but after finding that this worked, I added it for my own muscle memory.  I'll defer to you and @sjrd.  Should I remove it?

Reply:
I have a slight preference for not having the command alias, and running `scalastyleCheck` instead. This is mostly because I don't like command aliases that shadow tasks that otherwise exist.

Reply:
Removed, and updated Jenkinsfile/etc to use `scalastyleCheck`.  Thanks!